app/router.go

```diff
@@ -75,8 +75,13 @@ func Router(c collector) *mux.Router {
 	get.MatcherFunc(URLMatcher("/api/origin/host/{id}")).HandlerFunc(
 		gzipHandler(makeOriginHostHandler(c)))
 	get.HandleFunc("/api/report", gzipHandler(makeRawReportHandler(c)))
+	get.HandleFunc("/api/controls", controlRouter.HandleProbeWS)
 	get.PathPrefix("/").Handler(http.FileServer(FS(false))) // everything else is static
 
+	post := router.Methods("POST").Subrouter()
+	post.HandleFunc("/api/report", makeReportPostHandler(c)).Methods("POST")
+	post.MatcherFunc(URLMatcher("/api/control/{probeID}/{nodeID}/{control}")).HandlerFunc(
+		controlRouter.HandleControl)
```

Comment:
Updated the prefixes (its /api/control/ws and /api/control/foo/bar/baz now).  Broken them out into a separate registration function.

The problem with PathPrefix & subrouters was two fold: (1) the path prefix from / static files was owning everything, and (2) path prefixes and our custom matcher funcs don't mix.  So we can't use PathPrefix here.